src/Paket.VisualStudio/SolutionExplorer/PaketMenuCommandService.cs

```diff
@@ -366,6 +370,45 @@ private void RemovePackage(object sender, EventArgs e)
             });
         }
 
+        private void AddPackage(object sender, EventArgs e)
+        {
+            RunCommandAndReloadAllProjects(sender, e, "paket-add.html", info =>
```

Comment:
I think I used the wrong command type. Since this not adding the package to any project we actually don't need to unload any project.